.env

```diff
@@ -0,0 +1,5 @@
+EVENT360_CLIENT_ID=3WdrqHNMYoZR2Oii5hYMVivC1LZFZPutfZM7dhXdNCo
+EVENT360_URL = https://event360-dev-eventnxt-152500be953e.herokuapp.com
+
+EVENT360_CLIENT_SECRET=MXNaOBUX0xRJTu1qTrRVquVXBL2y_qpsb22enUoN3Jg
```

Comment:
should not be committed to Github ideally

Reply:
yeah, I'll update it in later PRs

Reply:
this pair is not permanent, separate pair will be generated for that.

Reply:
ok. Also it should be `EVENT360_URL=https://event360-dev-eventnxt-152500be953e.herokuapp.com` right?